app/views/flash.erb

```diff
@@ -0,0 +1,17 @@
+
+<!-- All tests failed when partials were used. Needs working on -->
```

Comment:
well done for giving them a try - ideally any code like this that didn't work should be moved into another branch called, say, 'partials' ... then that won't clutter up your submission that doesn't involve them